test/mitmproxy/addons/test_onboarding.py

```diff
@@ -19,3 +20,13 @@ def test_cert(self):
                 resp = self.app("/cert/%s" % ext)
                 assert resp.status_code == 200
                 assert resp.content
+
+    def test_head(self):
+        with taddons.context() as tctx:
+            tctx.configure(self.addons()[0])
+            p = self.pathoc()
+            for ext in ["pem", "p12"]:
```

Comment:
Could you please refactor this loop to use [pytest.mark.parametrize](https://docs.pytest.org/en/latest/parametrize.html) instead (same for the existing test case)